src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/InfiniteScrollListBoxAutomationPeer.cs

```diff
@@ -26,8 +28,19 @@ protected override List<AutomationPeer> GetChildrenCore()
             {
                 await infiniteScrollListBox.ItemsLock.ExecuteAsync(() =>
                 {
-                    // Don't return the LoadingStatusIndicator as an AutomationPeer, otherwise narrator will report it as an item in the list of packages, even when not visible
-                    value = base.GetChildrenCore()?.Where(lbiap => !(((ListBoxItemAutomationPeer)lbiap).Item is LoadingStatusIndicator)).ToList() ?? new List<AutomationPeer>();
+                    List<AutomationPeer> items = base.GetChildrenCore();
+                    if (items == null)
+                    {
+                        value = new List<AutomationPeer>();
+                    }
+                    else
+                    {
+                        var listCollectionView = (ListCollectionView)CollectionViewSource.GetDefaultView(items);
+                        // Don't return the LoadingStatusIndicator as an AutomationPeer, otherwise narrator will report it as an item in the list of packages, even when not visible
+                        IEnumerable<AutomationPeer> packageItemsInView = items.Where(item => ((ListBoxItemAutomationPeer)item).Item is PackageItemListViewModel
+                                                                                             && listCollectionView.Contains(item));
```

Comment:
format: The formatting seems off here. The operators are usually kept on the previous line, but the main thing that stands out is the extra whitespace compared to the line above.

Reply:
Can you use the suggestion feature? I don't know what our standard is for this.

Reply:
I don't think we have it documented explicitly, cause I think it's a bit difficult to verbalize. 

Here's an example: https://github.com/NuGet/NuGet.Client/blob/3544552bd438887c3ba221b73353db47b61cc8fa/src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreRunner.cs#L192-L196

I'll try use the suggestion to see what comes out.

Reply:
Seems like we want.... new line with 2 tabs?

Reply:
How's it look now?

Reply:
Looks better.